lib/splitclient-rb/cache/adapters/memory_adapters/queue_adapter.rb

```diff
@@ -10,6 +10,11 @@ def initialize(max_size)
             @current_size = Concurrent::AtomicFixnum.new(0)
           end
 
+          def clear(_ = nil)
+            @queue = Queue.new
+            @current_size = Concurrent::AtomicFixnum.new(0)
```

Comment:
why are you assigning a new variable? if the existing variable is atomic, why not setting it to zero?